fendermint/actors/bucket/src/state.rs

```diff
@@ -215,7 +190,58 @@ impl State {
         })?;
 
         let common_prefixes = common_prefixes.into_iter().collect();
-        Ok((common_prefixes, next_key))
+        Ok((common_prefixes, next_key.map(|key| key.0)))
+    }
+}
+
+#[derive(Debug, PartialEq)]
+pub struct ObjectKey(pub BytesKey);
```

Comment:
This is a bit annoying, but I had to do it. `BytesKey` does not implement `Display` which is needed, and we can't implement `Display` on `BytesKey` because of the orphan rule